File: 151002/Kuvshinov/lab1/Models/Story.cs
namespace DC.Models
{
	public class Story
	{
		public long Id { get; set; }

		public long EditorId { get; set; }

		public Editor Editor { get; set; }

		public string Title { get; set; }

		public string Content { get; set; }

		public DateTime Created { get; set; }

		public DateTime Modified { get; set; }
	}
}

File: 151001/Tarabarova/lab1/src/main/java/com/example/lab1/Model/Story.java
package com.example.lab1.Model;

import jakarta.validation.constraints.Size;
import lombok.Data;

import java.security.Timestamp;

@Data
public class Story {
    int id;
    int editorId;
    @Size(min = 2, max = 64)
    String title;
    @Size(min = 4, max = 2048)
    String content;
 //   Timestamp created;
  //  Timestamp modified;

}
